### PR TITLE
reformat markdown pages

### PR DESCRIPTION
**Pull request by v-kents**

### CODE REVIEW

**Comment by v-kents** [comment]
@cmcclister One to review, please (and thank you)

File: spec/standard-library.md [hunk 1]
```diff
@@ -1,730 +1,535 @@
...
-
-namespace System\
-{\
-public class ArgumentException : SystemException\
-{\
-public ArgumentException();\
-public ArgumentException(string message);\
-public ArgumentException(string message, Exception innerException);\
-}\
+namespace System {
```
Comment (lindexi) [inline spec/standard-library.md]: In [corefx/coding-style.md at master · dotnet/corefx](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md ) NO.1, we use Allman style braces, where each brace begins on a new line. A single line statement block can go without braces but the block must be properly indented on its own line and must not be nested in other statement blocks that use braces (See issue 381 for examples). One exception is that a using statement is permitted to be nested within another using statement by starting on the following line at the same indentation level, even if the nested using contains a controlled block."

File: spec/standard-library.md [hunk 2]
```diff
@@ -1,730 +1,535 @@
-# Standard library {#standard-library .Appendix1}
+# Standard library
+
+## General
+
+A conforming C# implementation shall provide a minimum set of types having specific semantics. These types and their members are listed here, in alphabetical order by namespace and type. For a formal definition of these types and their members, refer to ISO/IEC 23271:2012 *Common Language Infrastructure (CLI), Partition IV; Base Class Library (BCL), Extended Numerics Library, and Extended Array Library*, which are included by reference in this specification.
+
+> [!NOTE]
```
Comment (cmcclister) [inline spec/standard-library.md]: @v-kents was there a Note around the informative text? Please see [https://review.docs.microsoft.com/en-us/content-production-service/internal/c-sharp-language-guide?branch=c-sharp-task#normative-and-informative-text](https://review.docs.microsoft.com/en-us/content-production-service/internal/c-sharp-language-guide?branch=c-sharp-task#normative-and-informative-text). Please also see lindexi comment: [https://github.com/dotnet/csharplang/pull/3136/files#r373770237](https://github.com/dotnet/csharplang/pull/3136/files#r373770237). Once these are resolved, then I believe that this PR is ready for Bill Wagner to review. Nice work!

Comment (v-kents) [reply spec/standard-library.md]: I was going off of Any instance of the above should at minimum be indicated with a blockquote (> on each line of the informative text), with the, so I guess it needs the >, but no NOTE header?

**Pull request merged**